src/asm/asmy.y

```diff
@@ -1313,6 +1313,12 @@ string		: T_STRING
 		{
 			strcpy($$, $1);
 		}
+		| T_STRING T_STRING
```

Comment:
If this C-style `"string" "concatenation"` syntax isn't fully working (with the one-character example you gave), and the functionality is already provided by `STRCAT`, there's probably not very much point in including this.

I'm also not aware that this has been requested, but I'm happy to be overruled if this is something that would be helpful for some people!

Reply:
The problem is that one character long strings are considered a const. C solves this by having single quotes for characters and double quotes for strings, but rgbasm doesn't have this... If I want to support that, even `"hello " 32 "world"` would be valid.

The best thing I can do is to remove it, I think.